nes/config.py

```
@@ -92,19 +112,22 @@ def ensure_db_path_exists(cls, db_path: Optional[Path] = None) -> Path:
         return db_path
 
     @classmethod
-    def initialize_database(cls, base_path: str = "./nes-db/v2") -> "EntityDatabase":
+    def initialize_database(cls, base_path: Optional[str] = None) -> "EntityDatabase":
```

Flake8 (7.3.0)

[error] undefined name 'EntityDatabase'

(F821)